packages/accordion/src/LionAccordion.js

```diff
@@ -180,12 +202,33 @@ export class LionAccordion extends LitElement {
       };
       this._setupContent(entry);
       this._setupInvoker(entry);
+      this.__rearrangeInvokersAndContent();
       this._unfocusInvoker(entry);
       this._collapse(entry);
       this.__store.push(entry);
     });
   }
 
+  /**
+   *  @private
+   */
+  __rearrangeInvokersAndContent() {
+    const invokers = /** @type {HTMLElement[]} */ (
+      Array.from(this.querySelectorAll('[slot="invoker"]'))
+    );
+    const contents = /** @type {HTMLElement[]} */ (
+      Array.from(this.querySelectorAll('[slot="content"]'))
+    );
+    const accordion = this.shadowRoot?.querySelector('slot[name=accordion]');
```

Comment:
why would/could this be null? Maybe you can do `/** @type {ShadowRoot} */ (this.shadowRoot)`

Reply:
I can but I followed the convention in this file which is currently `this.shadowRoot?`.